bundler.d/test.rb

```diff
@@ -18,7 +18,7 @@
   gem 'rubocop-rails', '~> 2.4.2'
   gem 'poltergeist', '>= 1.18.0', :require => false
   gem 'selenium-webdriver', :require => false
-  gem 'shoulda-matchers', '~> 4.0'
+  gem 'shoulda-matchers', '~> 4.0', '< 4.4'
```

Comment:
The script that converts this to RPM deps creates something ugly from this. It creates `< 5.0` and `< 4.4`.
```suggestion
  gem 'shoulda-matchers', '>= 4.0', '< 4.4'
```

Reply:
didn't think we created rpms for test gems, but no reason to not doing it the better way for copy/paste reasons :)

Reply:
Ah, we indeed skip it for test gems so it probably wouldn't have mattered.